var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -50,19 +50,19 @@ class PyJaxlib(PythonPackage, CudaPackage):
     def patch(self):
         self.tmp_path = tempfile.mkdtemp(prefix="spack")
         self.buildtmp = tempfile.mkdtemp(prefix="spack")
```

Comment:
This looks weird to me. Can it be in the stage dir? `mkdtemp(dir=...)`.

Reply:
Let's do that in a different PR. We aren't currently consistent about the build directory for bazel packages. The problem is that bazel doesn't work on NFS (and possibly other parallel file systems?) so it has to be done locally, which is why we usually use `/tmp` instead of the default build stage. But `/tmp` is often small so users have to override env vars to build elsewhere, which is annoying. Would be nice if we could instead check if the default build stage is NFS or not and only change when that's the case.